espflash/src/targets/esp32p4.rs

```diff
@@ -20,7 +20,7 @@ const FLASH_RANGES: &[Range<u32>] = &[
 const PARAMS: Esp32Params = Esp32Params::new(
     0x2000,
     0x1_0000,
-    0x3f_0000, // TODO: Update
+    0x3f_0000,
```

Comment:
Not sure why I had added this originally, seems to work fine for now so just removed the comment.